code/game/gamemodes/blob/theblob.dm

```diff
@@ -196,6 +198,11 @@
 		color = a_color
 	return
 
+/obj/effect/blob/proc/update_desc(var/a_desc=null)

```

Comment:
Not without changing how blob works a bit.  Blob parts(except the core) dont track their origin/ mind/reagent at all so this needs to be updated when it changes type/other blob affects it/chem rerolls